deck.gl/docs/guides/01-getting-started.md

```diff
@@ -1,205 +1,49 @@
 
-In this guide, you will learn the basics of visualizing data with the Web SDK on top of a world [basemap](https://carto.com/help/glossary/#basemap). There are no previous requirements to complete this guide, but a basic knowledge of HTML, CSS and JavaScript would be helpful.
+In this guide, you will learn how to install CARTO for deck.gl in your computer and run the examples.
 
-After completing this guide, you will have your first Web SDK map!
+### Installation
 
-<div class="example-map">
-    <iframe
-        id="getting-started-final-result"
-        src="/developers/web-sdk/examples/maps/guides/getting-started/step-3.html"
-        width="100%"
-        height="500"
-        frameBorder="0">
-    </iframe>
-</div>
+You can work with CARTO for deck.gl with the NPM module for React or Pure JS applications:
 
-### Basic setup
+`npm install @deck.gl`
 
-The most straight-forward way to use the Web SDK is to include the required files from our CDN as seen in the code below. The Web SDK is based on the powerful deck.gl library. In addition to the Web SDK JavaScript file, you need to add:
+`npm install @deck.gl/carto`
 
-* deck.gl JavaScript file
-* Mapbox GL JavaScript and CSS files
+Or you can use the scripting API:
 
-```html
-    <!-- Include deck.gl from unpkg CDN -->
-    <script src="https://unpkg.com/deck.gl@8.2.0/dist.min.js"></script>
+`<script src="https://unpkg.com/deck.gl@latest/dist.min.js"></script>`
 
-    <!-- Include Mapbox GL from the Mabpox CDN -->
-    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js'></script>
-    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css' rel='stylesheet' />
+`<script src="https://unpkg.com/deck.gl@latest/carto/dist.min.js"></script>`
 
-    <!-- Include Web SDK from the CARTO CDN -->
-    <script src="https://libs.cartocdn.com/web-sdk/%VERSION%/index.min.js"></script>
-```
+You can learn more about deck.gl on the official [website](https://deck.gl), where you can find a dedicated section on the @deck.gl/carto [submodule](https://deck.gl/docs/api-reference/carto/overview).
 
-**Note:**
-**deck.gl and Mapbox GL**: The Web SDK is compatible with specific deck.gl and Mapbox GL versions. We recommend using the same versions that we use in the [examples](/developers/web-sdk/examples/). 
+### Examples
 
-**Note:**
-**Developers**: if you have experience with `npm` and a build system (_webpack_, _rollup_...), you can install the Web SDK in your project with `npm install @carto/web-sdk` and import it with `import carto from '@carto/web-sdk'`.
+In the [Examples]({{site.baseurl}}/deck-gl/examples) section you can find interactive examples using the scripting API. Here we show how you can download and run the React and PureJS examples. These are small, standalone examples that could be a good starting point for your application showing how to add different CARTO layers, how to integrate in a React application or how to use it with webpack and PureJS.
```

Comment:
probably 'pure JS' instead of PureJS, as that suggest a library, instead of vanilla JavaScript

Reply:
Changed